### PR TITLE
Content changes for TeacherCon

### PR DESCRIPTION
**Pull request by joshlory**

### CODE REVIEW

File: dashboard/config/scripts/u1l4_lesson_overview.external [hunk 1]
```diff
@@ -9,9 +9,9 @@ markdown <<MARKDOWN
...
+Students will explore the properties of number systems by effectively inventing a base-3 number system using circles, triangles and squares as the symbols instead of arabic numerals. Students are asked to create rules that explain how each arrangement of symbols can be generated or predicated as an orderly, logical series. The objective is to understand that you can represent any number with any agreed-upon set of symbols that appear in an agreed-upon order. This is as true for circles, triangles and squares as it is for the digits 0-9, or the number systems we commonly see in computer science (binary and hexadecimal).
```
Comment (mehalshah) [inline dashboard/config/scripts/u1l4_lesson_overview.external:13]: Looks like we don't like oxford commas

File: dashboard/config/scripts/levels/Wednesday Report: Group Work and Peer Learning in Unit 1.level [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <FreeResponse>
   <config><![CDATA[{
   "game_id": 52,
-  "created_at": "2016-06-15T10:29:37.884Z",
+  "created_at": "2016-06-15T10:29:37.000Z",
```
Comment (mehalshah) [inline dashboard/config/scripts/levels/Wednesday Report: Group Work and Peer Learning in Unit 1.level:4]: Weird that this is changing?

**Comment by mehalshah** [comment]
lgtm

File: dashboard/config/scripts/u2l2_lesson_overview.external [hunk 3]
```diff
@@ -16,21 +16,19 @@ At some point we reach a physical limit of how fast we can send bits and if we w
...
+- This is a big multi-day lesson.  At TeacherCon your goal is to get through the first activity &mdash; using the text compression widget (you **do not** have to get to “developing a heuristic” but it’s conceivable you could foreshadow it).
 - At TeacherCon you might want to shorten or omit one of the two warm up activities.  Discuss with your group and mentors the benefits and drawbacks of these choices.  It’s also possible to do both and to cut off the activity.
 - Of course we want participants engaged with the widget as soon as possible, but do take care to make sure that they know what they’re doing before getting on the computer.  
 - A big decision to make with this lesson is whether to show the tutorial video “text compression with Aloe Blacc” to motivate the lesson at the outset - it’s also appropriate to show as a wrap up, the choice is yours.  You also need to decide whether to use the tutorial video to show how to use the widget or whether you will demonstrate it more interactively - a live demo is almost always more engaging, but be very wary of how much time it takes.
```
Comment (mehalshah) [inline dashboard/config/scripts/u2l2_lesson_overview.external]: Why does line 19 get an em-dash and 22 does not?

Comment (joshlory) [inline dashboard/config/scripts/u2l2_lesson_overview.external]: I'll add it.

**Comment by mehalshah** [comment]
One change, the rest looks fine

**Pull request merged**